server/views/components/hearing-notes/template.njk

```diff
@@ -0,0 +1,28 @@
+<div class="app-summary-card__body">
+
+    <div class="govuk-grid-row govuk-grid-column-full app-summary-card__body">
+        {% for previousNote in params.hearing.notes %}
+            <span class="govuk-!-margin-bottom-2">{{ previousNote.note }}</span>
+            <span class="govuk-caption-m govuk-!-margin-top-2 govuk-!-font-size-16">{{ previousNote.author }} on {{ previousNote.noteCreated | caseCommentTimeFormat }}
+                            <a class="govuk-link govuk-!-margin-left-5" href="#">Delete</a>
+                        </span>
+            <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+        {% endfor %}
+    </div>
+
+    <form id="save-notes-{{ params.hearing.hearingId }}" action="summary/notes" method="post">
+
+        <details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details">
+            <summary class="govuk-details__summary govuk-!-margin-left-2">
+                <span class="govuk-details__summary-text">Add a note</span>
+            </summary>
+            <div class="govuk-details__text">
```

Comment:
```suggestion
            <div>
```

Reply:
Above removes the left border which is not a requirement for add note 
<img width="543" alt="image" src="https://user-images.githubusercontent.com/98027245/192584134-2f4e9968-01be-4861-aa58-faf6333c112f.png">

Reply:
I will verify with the designers, but I think there is a mistake on this when you look at PIC-2370. It's seems to show 2 different versions of this box

Reply:
After verification with Jo, the right component is the one with the left border.